site/docs/api/sources-sinks.md

```diff
@@ -899,6 +899,26 @@ p.readFrom(Sources.remoteMap("inputMap", cfg));
 ...
 ```
 
+#### Compatibility
+
+When reading or writing to remote sources, Jet internally creates a
+client. This client uses the embedded IMDG version to connect to the
+remote cluster. Starting with Hazelcast 3.6, Hazelcast server & client
+versions are backward and forward compatible within the same major
+version.
+
+|Jet Version|Embedded IMDG Version|CompatibleVersions|
+|:-----|:------------------|:-----------|
+|Jet 3.0|Hazelcast 3.12|Hazelcast 3.x.x|
+|Jet 3.1    |Hazelcast 3.12.1   |Hazelcast 3.x.x|
+|Jet 3.2    |Hazelcast 3.12.3   |Hazelcast 3.x.x|
+|Jet 3.2.1  |Hazelcast 3.12.5   |Hazelcast 3.x.x|
+|Jet 3.2.2  |Hazelcast 3.12.6   |Hazelcast 3.x.x|
+|Jet 4.0    |Hazelcast 4.0      |Hazelcast 4.x.x|
```

Comment:
`Hazelcast 4.x.x` looks quite weird for me, should not we use just `4.x`? or at least `4.x.y` or `4.y.z`?